xbmc/cores/VideoPlayer/VideoPlayerRadioRDS.cpp

```diff
@@ -1082,11 +1025,7 @@ inline void rtrim_str(std::string &text)
 
 unsigned int CDVDRadioRDSData::DecodeRT(uint8_t *msgElement, unsigned int len)
 {
-  if (!m_RT_Present)
-  {
-    m_currentInfoTag->SetPlayingRadiotext(true);
-    m_RT_Present = true;
-  }
+  m_currentInfoTag->SetPlayingRadiotext(true);
```

Comment:
Capital `T` for `Text`?

I know this is from the existing code, but not reason not to fix it while your there.

Reply:
Changed.